githubdelivery/src/main/java/io/dockstore/githubdelivery/GithubDeliveryS3Client.java

```diff
@@ -181,15 +181,15 @@ private void submitGitHubDeliveryEventsByKey(String key, WorkflowsApi workflowsA
         try {
             String body = getObject(key);
             JsonObject jsonObject = GSON.fromJson(body, JsonObject.class);
-            if (jsonObject.get("action").getAsString().equals("added") || jsonObject.get("action").getAsString().equals("removed")) {
+            if (jsonObject.get("action") != null) { //Instatllation and uninstallation events have action property
```

Comment:
Just to be clear, are "added" and "removed"  (installation and uninstall events) the only possible values here? 
i.e. the old version would skip events that were not-null but neither of the two, the new version would process them